source/_integrations/adax.markdown

```diff
@@ -18,14 +18,18 @@ Integrates Adax heater into Home Assistant.
 
 You can configure the heaters to use a local or cloud interface.
 
-The local integrations only works with newer Adax heaters with both Bluetooth and wifi. Home Assistant uses Bluetooth to configure the heaters. For the local integration, you have to reset the heater by pressing + and OK until the display shows 'Reset'. Then press and hold the OK button on the heater until the blue LED starts blinking before pressing Submit. Configuring the heater might take some minutes. Using the local integration will disable cloud communication and the Adax app will not work.
+The local integrations only works with newer Adax heaters with both Bluetooth and wifi. Home Assistant uses Bluetooth LE to configure the heaters, this means your machine running “Home Assistant” needs to have a Bluetooth adapter and the heater needs to be in range. For the local integration, you have to reset the heater by pressing + and OK until the display shows 'Reset'. Then press and hold the OK button on the heater until the blue LED starts blinking before pressing Submit. Configuring the heater might take some minutes. Using the local integration will disable cloud communication and the Adax app will not work.
```

Comment:
Does it? What about BT proxies?

Reply:
Bluetooth proxies bridge Bluetooth LE over wifi. Here BLE is used to push AP config to the heater (that has wifi and thus does not need a BT proxy).

Reply:
No luck with the bluetooth proxy. It just does not seem to detect the heater (while I can see it fine from my phone or laptop). It does not seem to be a proxy issue either as it does pick up one of my neighbours BLE beacon